lib/r10k/puppetfile.rb

```diff
@@ -249,6 +344,16 @@ def moduledir(location)
       @librarian.set_moduledir(location)
     end
 
+    def incl(librarian)
+      if Pathname.new(librarian).absolute?
+        path = arg
+      else
+        path = File.join(File.dirname(@librarian.puppetfile_path), librarian)
+      end
+
+      instance_eval(File.read(path), @librarian.puppetfile_path)
```

Comment:
If we're trying to avoid eval'ing code directly then this line breaks that goal.